src/CMakeLists.txt

```
@@ -48,10 +56,27 @@ set_target_properties(podioDict-dictgen PROPERTIES EXCLUDE_FROM_ALL TRUE)
 
 target_sources(podioDict PRIVATE podioDict.cxx)
 
-# Install the Targets and Headers
-install(TARGETS podio podioDict podioRootIO
-  EXPORT podioTargets
-  DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+# SIO I/O library
+if(SIO_HANDLERS)
+add_library(podioSioIO SHARED ${sio_sources})
+add_library(podio::podioSioIO ALIAS podioSioIO)
+target_include_directories(podio PUBLIC
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+target_link_libraries(podioSioIO PUBLIC podio::podio ${SIO_LIBRARIES})
+
+  # Install the Targets and Headers
+  install(TARGETS podio podioDict podioRootIO podioSioIO
+    EXPORT podioTargets
+    DESTINATION "${CMAKE_INSTALL_LIBDIR}")
+else()
+  # Install the Targets and Headers
+  install(TARGETS podio podioDict podioRootIO
```

```suggestion
  install(TARGETS podio podioDict podioRootIO ${INSTALL_LIBRARIES}
```
Then you don't need different install commands, if the variable is not set it isn't a problem, but it scales to more IOHandlers easily.
I hope a list works here.

---

Just for future reference: a list seems to work nicely here.